src/doc/rustc/src/platform-support.md

```diff
@@ -254,6 +254,7 @@ target | std | host | notes
 [`aarch64-kmc-solid_asp3`](platform-support/kmc-solid.md) | ✓ |  | ARM64 SOLID with TOPPERS/ASP3
 [`aarch64-nintendo-switch-freestanding`](platform-support/aarch64-nintendo-switch-freestanding.md) | * |  | ARM64 Nintendo Switch, Horizon
 [`aarch64-unknown-teeos`](platform-support/aarch64-unknown-teeos.md) | ? |  | ARM64 TEEOS |
+[`aarch64-unknown-nto-qnx700`](platform-support/nto-qnx.md) | ✓ |  | ARM64 QNX Neutrino 7.0 RTOS |
```

Comment:
I don't agree with this change just `yet`: We don't have QNX 7.0 support in libc at the moment! Let's add support there first and then merge this change please.

Reply:
We should also keep `src/doc/rustc/src/platform-support/nto-qnx.md` in sync.

Reply:
@flba-eb this PR cannot merge without this line because there is a CI check that every target file is also listed here. I am waiting for a proper ssh tunnel into a QNX 7.0 workbench to do thorough testing - in the mean time i did a bunch of spot tests.

One unexpected stumbling block was that depending on QNX kernel configuration, the same binary could refuse to execute unless this was added to `.cargo/config.toml`:

```toml
[target.aarch64-unknown-nto-qnx700]
rustflags = ["-C", "relro_level=off"]
```

without this, it would fail with `syntax error: .... unexpected`.  Debugging the binary startup with `export DL_DEBUG=all` showed that the resolution scope was incorrect (it should have been resolving to the binary itself)

```
Resolution scope for Executable->/bin/sh:
        Executable->/bin/sh
        libc.so.4->/usr/lib/ldqnx-64.so.2
```

I do not think relro should be disabled by default, but I wonder if this info should be added to the `nto-qnx.md` -- it did take me a long time to figure it out.

Reply:
thx, updated